standalone-examples/src/main/java/io/pravega/example/binary/BinarySerializer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.binary;
+
+import io.pravega.client.stream.Serializer;
+import java.nio.ByteBuffer;
+
+public class BinarySerializer implements Serializer<ByteBuffer> {
+    @Override
+    public ByteBuffer serialize(ByteBuffer value) {
+        return value;
+    }
+
+    @Override
+    public ByteBuffer deserialize(ByteBuffer serializedValue) {
+        return serializedValue;
```

Comment:
Maybe the buffer should be copied because I'm unsure as to whether the buffer is reused on the next element.

Reply:
I disagree. The `ByteBuffer` is likely reused but the same segment of it gets read/written on each subsequent read/write. The lifecycle of the value that is passed in is the same as the buffer. This seems like a good optimization not having to make a copy of this each time.